integration/docker/Dockerfile

```diff
@@ -41,36 +27,19 @@ RUN cd /opt && \
     (if ls | grep -q ".tar.gz"; then tar -xzf *.tar.gz && rm *.tar.gz; fi) && \
     ln -s alluxio-* alluxio
 
-RUN if [ ${ENABLE_DYNAMIC_USER} = "true" ] ; then \
-       chmod -R 777 /opt/* ; \
-    fi
-
-# Configure Java
-FROM centos:7 as build_java8
-RUN \
-    yum update -y && yum upgrade -y && \
-    yum install -y java-1.8.0-openjdk-devel java-1.8.0-openjdk && \
-    yum clean all
-ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk
-# Disable JVM DNS cache in java8 (https://github.com/Alluxio/alluxio/pull/9452)
-RUN echo "networkaddress.cache.ttl=0" >> /usr/lib/jvm/java-1.8.0-openjdk/jre/lib/security/java.security
-
-FROM centos:7 as build_java11
+FROM centos:7 as final
 RUN \
     yum update -y && yum upgrade -y && \
     yum install -y java-11-openjdk-devel java-11-openjdk && \
-    yum clean all
+    yum clean all;
```

Comment:
You added a semicolon here. Is it necessary?

Reply:
I don't think so. Typo